src/test/CMakeLists.txt

```diff
@@ -507,9 +507,12 @@ add_dependencies(check
   cython_modules)
 
 add_ceph_test(test-ceph-helpers.sh ${CMAKE_CURRENT_SOURCE_DIR}/test-ceph-helpers.sh)
+add_ceph_test(ceph_objectstore_tool.py ${CMAKE_CURRENT_SOURCE_DIR}/ceph_objectstore_tool.py)
+set_property(TEST test-ceph-helpers.sh ceph_objectstore_tool.py
+  PROPERTY LABELS Racing LongRunning)
```

Comment:
@tchaikov where is the documentation for the "Racing" and "LongRunning" LABELS? I tried looking online and nothing popped up quickly. Seems like a cool feature!